alot/defaults/alot.rc.spec

```diff
@@ -248,6 +248,9 @@ reply_account_header_priority = force_list(default=list(From,To,Cc,Envelope-To,X
 #           history file might get *very* long.
 history_size = integer(default=50)
 
+# The number of seconds to wait between calls to the loop_hook
+loop_period = integer(default=300)
```

Comment:
I'm not a fan of the name: How about "periodic_hook_frequency"?
although technically this describes the period, not a frequency.
By the way. Are you sure that this is in seconds, not milliseconds?